tests/Library/Core/ArrayTranslateTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @author Patrick Kelly <patrick.k@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license Proprietary
```

Comment:
We've got the wrong license here.

```suggestion
 * @license GPL-2.0-only
```

Reply:
Go to your PHPStorm preferences and search for "File and Code Templates". You can edit the default header on a per project basis to include the correct header. I always miss the license too so I found the preference very useful.